server_src/routes/index.js

```diff
@@ -490,6 +494,13 @@ router.get("/system", ensureAuthenticated, async (req, res) => {
     systemInformation,
     db,
     dashboardSettings: dashboardSettings,
+    serviceInformation: {
```

Comment:
Add API smoke-test. Look at supertest usage (test already present in test folder)